### DISCUSSION TITLE
Radar chart does not display correctly on redash

### DISCUSSION THREAD

**Discussion by qq20230530** [getredash/redash#6189] [Q&A]
Found an example on codepen:
https://codepen.io/etpinard/pen/aPXozx?editors=1010

Copied this to redash, but radar chart looks strange, what might be the reason?

Custom code:
Plotly.newPlot(
  element,
  [
    {
      type: "scatterpolar",
      r: [39, 28, 8, 7, 28, 39],
      theta: ["A", "B", "C", "D", "E", "A"],
      fill: "toself",
      name: "Group A"
    },
    {
      type: "scatterpolar",
      r: [1.5, 10, 39, 31, 15, 1.5],
      theta: ["A", "B", "C", "D", "E", "A"],
      fill: "toself",
      name: "Group B"
    }
  ],
  {
    polar: {
      radialaxis: {
        visible: true,
        range: [0, 50]
      }
    },
    showlegend: false
  }
);

Displayed radar chart:
![image](https://github.com/getredash/redash/assets/135001174/e047e07d-4925-4244-8461-85f7f335df6b)

**Comment by gaecoli** [comment]
Maybe you should change `r` to `ys`?

**Comment by gaecoli** [comment]
I also tried to draw a radar chart, it may be a version problem of plotlyjs, it has been unable to draw normally.

**Reply by justinclift** [reply]
Is that with our latest development version?

**Reply by gaecoli** [reply]
@qq20230530 are you?

**Reply by qq20230530** [reply]
I am using v10.0.0

**Comment by gaecoli** [comment]
I guess it may be caused by the version of our plotlyjs being too low.

**Reply by justinclift** [reply]
Do you have time and interest to try upgrading Plotly on your local computer to see if newer version works better?

**Reply by gaecoli** [reply]
OK.

**Reply by gaecoli** [reply]
I tried it, not work for me.

**Reply by justinclift** [reply]
Thanks.  So that is probably not the cause then.  It will take some more investigation then. :smile:

**Reply by gaecoli** [reply]
Yeah! If we have time to do that!

**Reply by qq20230530** [reply]
Is there anything I can help to investigate?
